### PR TITLE
Add ApiKey to Ramp 

### PR DESCRIPTION
**Pull request by Sajma27**
# Summary
A description of the changes proposed in the pull request.

Fixes #number

## Checklist
- [ ] Change is small and easy to review (please split big changes into multiple PRs)
- [ ] Change is consistent with architecture
- [ ] Change is consistent with test architecture
- [ ] Change is consistent with naming conventions
- [ ] New code is covered with tests
- [ ] Tests related to old code are updated
- [ ] Documentation is up to date with changes

### CODE REVIEW

File: universal-login-wallet/src/config/config.ts [hunk 1]
```diff
@@ -25,5 +25,6 @@ export default Object.freeze({
     relayerUrl: process.env.RELAYER_URL!,
     jsonRpcUrl: process.env.JSON_RPC_URL!,
     tokens: [ETHER_NATIVE_TOKEN.address, process.env.DAI_TOKEN_ADDRESS!, process.env.JRT_TOKEN_ADDRESS!],
+    rampUrl: process.env.RAMP_URL,
```
Comment (JustynaBroniszewska) [inline universal-login-wallet/src/config/config.ts:28]: rename to `rampApiKey`

File: universal-login-commons/src/core/models/onRamp.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ export interface RampConfig {
   appName: string;
   logoUrl: string;
   rampUrl: string;
+  rampApiKey: string;
```
Comment (xitronix) [inline universal-login-commons/src/core/models/onRamp.ts:16]:
```suggestion
  rampApiKey?: string;
```

File: universal-login-wallet/src/config/config.ts (hunk 1 above)
Comment (JustynaBroniszewska) [reply universal-login-wallet/src/config/config.ts:28]: @Sajma27

File: universal-login-wallet/src/ui/createServices.ts [hunk 3]
```diff
@@ -11,6 +11,7 @@ export interface Config {
   relayerUrl: string;
   jsonRpcUrl: string;
   tokens: string[];
+  rampApiKey?: string;
```
Comment (xitronix) [inline universal-login-wallet/src/ui/createServices.ts:14]: Should be required here

Comment (Sajma27) [reply universal-login-wallet/src/ui/createServices.ts:14]: Are you sure? I think this config is used in tests where we don't have rampApiKey

File: universal-login-react/src/ui/TopUp/Fiat/TopUpWithFiat.tsx [hunk 4]
```diff
@@ -23,6 +23,9 @@ export interface TopUpWithFiatProps {
 }
 type TopUpWithFiatModal = 'none' | 'wait' | TopUpProvider;
 
+const getRampConfig = (config: RampConfig, rampApiKey?: string) =>
+  rampApiKey ? {...config, rampApiKey} : config;
+
```
Comment (xitronix) [inline universal-login-react/src/ui/TopUp/Fiat/TopUpWithFiat.tsx:28]: Move it to utils or Ramp.tsx

**Pull request merged**